kingpin/constants.py

```diff
@@ -46,3 +48,18 @@ class STATE(StringCompareBase):
     """
 
     valid = ('present', 'absent')
+
+
+class SchemaCompareBase(object):
```

Comment:
I probably should document this, and the StringCOmpareBase in the developer.rst doc file..